sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2737,9 +2772,15 @@ object GpuOverrides {
       "The backend for the sort operator",
       // The SortOrder TypeSig will govern what types can actually be used as sorting key data type.
       // The types below are allowed as inputs and outputs.
-      ExecChecks((TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL + TypeSig.ARRAY +
```

Comment:
The original code allowed ArrayType and nested structs to participate in the sort but not as a sort key.  This seems to remove that support?  Also shown in a loss of coverage in the change to `supported_ops.md`.  Did we not have tests verifying that before?

Reply:
Great catch I missed that.  We did not have tests verifying this before.  We should have.

Reply:
thanks for catching this @jlowe